docs/framework/windows-workflow-foundation/consuming-odata-feeds-from-a-workflow.md

```diff
@@ -77,24 +80,31 @@ WCF Data Services is a component of the [!INCLUDE[dnprdnshort](../../../includes
  [!code-csharp[CFX_WCFDataServicesActivityExample#20](../../../samples/snippets/csharp/VS_Snippets_CFX/CFX_WCFDataServicesActivityExample/cs/Program.cs#20)]  
   
  When this workflow is invoked, the following data is written to the console. Since this query returns many customers, only part of the output is displayed here.  
-  
- **Calling WCF Data Service...**  
-**Alfreds Futterkiste, Contact: Maria Anders**   
-**Ana Trujillo Emparedados y helados, Contact: Ana Trujillo**   
-**Antonio Moreno Taquería, Contact: Antonio Moreno**   
-**Around the Horn, Contact: Thomas Hardy**   
-**Berglunds snabbköp, Contact: Christina Berglund**   
-**...**    
+
+```
+Calling WCF Data Service...
+Alfreds Futterkiste, Contact: Maria Anders
+Ana Trujillo Emparedados y helados, Contact: Ana Trujillo
+Antonio Moreno Taquería, Contact: Antonio Moreno
+Around the Horn, Contact: Thomas Hardy
+Berglunds snabbköp, Contact: Christina Berglund
+...
+```
+
 ## Consuming an OData Feed Without Using the Client Libraries  
  OData exposes data as resources that are addressable by URIs. When you use the client libraries these URIs are created for you, but you do not have to use the client libraries. If desired, OData services can be accessed directly without using the client libraries. When not using the client libraries the location of the service and the desired data are specified by the URI and the results are returned in the response to the HTTP request. This raw data can then be processed or manipulated in the desired manner. One way to retrieve the results of an OData query is by using the <xref:System.Net.WebClient> class. In this example, the contact name for the customer represented by the key ALFKI is retrieved.  
   
  [!code-csharp[CFX_WCFDataServicesActivityExample#2](../../../samples/snippets/csharp/VS_Snippets_CFX/CFX_WCFDataServicesActivityExample/cs/Program.cs#2)]  
   
  When this code is run, the following output is displayed to the console:  
-  
- **Raw data returned:**  
-**\<?xml version="1.0" encoding="utf-8" standalone="yes"?>**   
-**\<ContactName xmlns="http://schemas.microsoft.com/ado/2007/08/dataservices">Maria Anders\</ContactName>**  In a workflow, the code from this example could be incorporated into the <xref:System.Activities.CodeActivity.Execute%2A> override of a <xref:System.Activities.CodeActivity>-based custom activity, but the same functionality can also be accomplished by using the <xref:System.Activities.Expressions.InvokeMethod%601> activity. The <xref:System.Activities.Expressions.InvokeMethod%601> activity enables workflow authors to invoke static and instance methods of a class, and also has an option to invoke the specified method asynchronously. In the following example, an <xref:System.Activities.Expressions.InvokeMethod%601> activity is configured to call the <xref:System.Net.WebClient.DownloadString%2A> method of the <xref:System.Net.WebClient> class and return a list of customers.  
+
+```xml
```

Comment:
same comment as above. 
This is mistagged as xml (at least, "Raw data returned:" isn't XML).